### PR TITLE
Add risk level banner for national escalation level

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard14**
Adds a banner that can accept a level and a date, in the future this should come from the data. 

Desktop:
Screenshot 2021-09-21 at 14 33 27

Mobile:
Screenshot 2021-09-21 at 14 34 49

### CODE REVIEW

File: packages/app/src/domain/escalation-level/escalation-level-banner.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+          <LinkWithIcon
+            href={text.banner.link.href}
+            icon={<ArrowIconRight />}
+            iconPlacement="right"
+            fontWeight="bold"
+          >
+            {text.banner.link.label}
+          </LinkWithIcon>
+        </Box>
```
Comment (VWSCoronaDashboard16) [inline packages/app/src/domain/escalation-level/escalation-level-banner.tsx:78]: It feels a bit much to render a completely different box on mobile, the layout is pretty similar. I think it's easier to just have 2 NL icons and hide or show them based on media.

File: packages/app/src/domain/escalation-level/logic/get-escalation-level.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { EscalationLevelType } from '~/domain/escalation-level/common';
+
+export function getEscalationLevelIndexKey(level: EscalationLevelType) {
```
Comment (VWSCoronaDashboard17) [inline packages/app/src/domain/escalation-level/logic/get-escalation-level.tsx:3]: This seems a little unnecessary?

**Review by VWSCoronaDashboard17 — APPROVED** [review]

File: packages/app/src/utils/use-escalation-level.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    assert(
+      escalationColor,
+      `Cannot resolve an escalation color for level '${level}'`
+    );
+
+    return {
+      color: escalationColor,
+      title:
+        siteText.national_escalation_levels.types[
+          getEscalationLevelIndexKey(level)
```
Comment (VWSCoronaDashboard16) [inline packages/app/src/utils/use-escalation-level.tsx:27]: Looking at this function, wouldn't a `level.toString()` to exactly the same thing?

File: packages/app/src/pages/landelijk/intensive-care-opnames.tsx [hunk 4]
```diff
@@ -241,7 +241,7 @@ const IntakeIntensiveCare = (props: StaticProps<typeof getStaticProps>) => {
                 }}
                 seriesConfig={[
                   {
-                    type: 'area',
+                    type: 'gapped-stacked-area',
```
Comment (VWSCoronaDashboard16) [inline packages/app/src/pages/landelijk/intensive-care-opnames.tsx:244]: What's this about?

**Review by VWSCoronaDashboard16 — CHANGES_REQUESTED** [review]

File: packages/app/src/domain/escalation-level/logic/get-escalation-level.tsx (hunk 2 above)
Comment (VWSCoronaDashboard1) [reply packages/app/src/domain/escalation-level/logic/get-escalation-level.tsx:3]: I'm guessing this is done to keep the compiler happy

File: packages/app/src/pages/landelijk/intensive-care-opnames.tsx (hunk 4 above)
Comment (VWSCoronaDashboard1) [reply packages/app/src/pages/landelijk/intensive-care-opnames.tsx:244]: Hehe, well spotted, this seems unrelated to this PR

**Review by VWSCoronaDashboard1 — APPROVED** [review]

File: packages/app/src/pages/landelijk/intensive-care-opnames.tsx (hunk 4 above)
Comment (VWSCoronaDashboard14) [reply packages/app/src/pages/landelijk/intensive-care-opnames.tsx:244]: Ah yeah, that was a mistake trying something not related to this PR.

File: packages/app/src/utils/use-escalation-level.tsx (hunk 3 above)
Comment (VWSCoronaDashboard14) [reply packages/app/src/utils/use-escalation-level.tsx:27]: `toString()` doesn't do much, but apparently ``${level}`` does the trick and doesn't get the compiler angry. But I'm not totally sure why that works but toString doesn't.

**Pull request merged**